coreos-assembler

```diff
@@ -4,14 +4,19 @@ set -euo pipefail
 # Currently this just wraps the two binaries we have today
 # under a global entrypoint with subcommands.
 
-# docker/podman don't run through PAM, but we want this set
+# docker/podman don't run through PAM, but we want this set for the privileged
+# (non-virtualized) path
 export USER=${USER:-$(id -nu)}
 
 # When trying to connect to libvirt we get "Failed to find user record
 # for uid" errors if there is no entry for our UID in /etc/passwd.
 # This was taken from 'Support Arbitrary User IDs' section of:
 #   https://docs.openshift.com/container-platform/3.10/creating_images/guidelines.html
 if ! whoami &> /dev/null; then
+  # We need to make sure we set $HOME in the /etc/passwd file because
+  # if we don't libvirt will try to use `/` and we will get permission
+  # issues
+  export HOME="/var/tmp/${USER_NAME:-default}" && mkdir -p $HOME
```

Comment:
@dustymabe I slightly modified your patch here to put it in `/var/tmp` instead of `/tmp` since the former is less likely to be on tmpfs.

Reply:
sounds good